lucien-ios/AddAndEditNewComicBook/ComicFormViewController.swift

```diff
@@ -311,6 +332,41 @@ final class ComicFormViewController: UIViewController, AlertDisplaying {
         finishButton.tintColor = LucienTheme.finishButtonGrey
     }
 
+    private func goToDashboard() {
+        let alertController = UIAlertController(title: "Error",
+                                                message: "Our service is currently encountering an issue. Please ensure that you are connected to the internet and try again.",
+                                                preferredStyle: .alert)
+        let okAction = UIAlertAction(title: "Ok", style: .cancel) { _ in }
+        alertController.addAction(okAction)
+        var dashboardComics = [DashboardComicUser]()
+        lucienAPIClient.getMyComics { response in
+            switch response {
+            case .success(let result):
+                DispatchQueue.main.async {
+                    dashboardComics = result
+                    self.lucienAPIClient.getDashboard { response in
+                        switch response {
+                        case .success(let result):
+                            let viewModel = DashboardViewModel(dashboard: result, myComics: dashboardComics)
+                            let dashboardViewController = DashboardViewController(dashboardViewModel: viewModel)
+                            let dashboardNavigationController = UINavigationController(rootViewController: dashboardViewController)
+                            self.present(dashboardNavigationController, animated: true, completion: nil)
+                        case .failure:
+                            self.present(alertController, animated: true, completion: nil)
+                        }
+                    }
+                }
+            case .failure:
+                self.present(alertController, animated: true, completion: nil)
+            }
+        }
+    }
+
+    private func dismissKeyboardAndNotifications() {
+        view.endEditing(true)
+        deregisterFromKeyboardNotifications()
+    }
+    
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Trailing Whitespace Violation: Lines should not have trailing whitespace. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.intrepid.io/coding_rules#rule_key=SwiftLint%3Atrailing_whitespace)